evaluation/main_nlu_prompt_batch.py

```diff
@@ -75,7 +84,9 @@ def to_prompt(input, prompt, labels, prompt_lang, schema):
     # Set seed before initializing model.
     set_seed(42)
 
-    model_runner = load_model_runner(MODEL)
+    # model_runner = load_model_runner(MODEL)
```

Comment:
Remove commented code and leave only new code

Reply:
Thank you for your contribution! Let's work on adding this new feature to the project.

Here are my comments:
1. Please clean up the existing comments in the code.
2. Move the run instructions to a separate file.
3. Add instructions on how to run the project on an OpenAI-compatible server in the README. (Also, please include a note that since OpenAI-compatible servers cannot access logits, the results on NLU tasks may not be directly comparable to the official leaderboard.)

Reply:
Regarding the third topic, fortunately, 
TensorRT-LLM can respond with both context logits and generation logits.

So, I will validate the result before committing again.